Server/Views/BaseServerView.cs

```diff
@@ -140,9 +172,32 @@ class ContextWrapper {
 					}
 					sb.Append(")");
 				}
+				var desc = build.Value.ShortDescription;
+				if ( !string.IsNullOrEmpty(desc) ) {
+					sb.Append(" - ");
+					sb.Append(desc);
+				}
 				sb.Append("\n");
 			}
 		}
+
+		void AppendBuildArgsInfo(StringBuilder sb, Build build) {
+			sb.Append("*Arguments list:*\n");
+			if ( build.Args == null || build.Args.Count == 0 ) {
+				sb.Append("No arguments.");
+				return;
+			}
+			var outputDescriptions = build.ArgsDescription.Count == build.Args.Count;
```

Comment:
Looks strange, cases, when description count is less can happen (e.g. new arg added, but without description, it such cases description for all args disappear).